clients/ts/signalr/src/HubConnection.ts

```diff
@@ -538,4 +594,27 @@ export class HubConnection {
             type: MessageType.CancelInvocation,
         };
     }
+
+    private createStreamDataMessage(id: string, item: any): StreamDataMessage {
```

Comment:
Remind me why we need this over a stream item?

Reply:
Stream Id